blog/2024-02-09-see-you-later-mr-lightning.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: see-you-later-mr-lightning
+title: See you later, Mr Lightning
+authors: [lucas]
+tags: [DLC-Channels, Lightning]
+---
+
+<!-- LTeX: SETTINGS language=en-GB-->
+
+If you are reading this post, you may have already heard that we have closed all Lightning channels with our users, replacing them with DLC channels.
+That may sound like a step backwards, so we want to explain our reasoning behind this decision and tell you why we think it is actually a good thing.
+
+## Why we started with Lightning
+
+10101 was conceived as a Lightning mobile wallet with trading capabilities.
+We think most bitcoiners have heard about Lightning, and they have probably even used a Lightning wallet before.
+It made sense to us to package our offering together with something that our target user was already comfortable with.
+
+And we saw some synergies between Lightning and DLCs, the technology that we have been using for years to help bitcoiners trade.
+We wanted our users to be onboarded instantly: generate a Lightning invoice; pay it with their own Lightning wallet; receive the funds instantly in their 10101 app; and start trading straight away.
+The user would then be able to move funds seamlessly between their trading and Lightning wallets.
+We even had ambitious plans to let users pay Lightning invoices with a synthetic US dollar stable coin, backed by a DLC.
+
+And we actually managed to do all of that. But the user experience wasn't nearly good enough.
+
+## Why Lightning was slowing us down
+
+We've had problems with Lightning at every step of the user's journey through 10101:
+
+- Users found it difficult to move coins into 10101.
+A Lightning payment can fail for many reasons, and it is often hard to pinpoint the cause.
+We generally overcame these issues as we learnt more about Lightning, but it is pretty scary to lose a potential user at the first hurdle.
+
+- Users found it very difficult to move coins out of 10101.
+If you have used 10101 to pay an invoice, you have almost certainly run into the dreaded `route not found` error message.
+We were working on solutions to this for
+
+- Channels would be force-closed for no apparent reason.
+Of course, there is always a reason.
+Disagreements during fee negotiations and expired HTLCs were the main culprits here.
+Regardless, any unnecessary force-closure is catastrophic because it is a waste of money and time for everyone involved.
+And this is particularly egregious when you are in a high transaction fee environment, like the one we experienced in 2023.
+
```

Comment:
Its probably worthwhile to add the huge on-chain footprint and complexity coming from an embedded dlc channel. The key promise of simply force closing a channel and you get your funds back wasn't that present anymore given high fee environments, etc.